packages/moonstone/Button/Button.js

```diff
@@ -50,7 +51,7 @@ const ButtonBase = kind({
 		 * * `'lightTranslucent'`, and
 		 * * `'transparent'`.
 		 *
-		 * @type {String}
+		 * @type {translucent'|'lightTranslucent'|'transparent'}
```

Comment:
Minor typo here, should be `@type {'translucent'|'lightTranslucent'|'transparent'}`

Reply:
Other examples in the framework show these being in `()`. Should this one also be? (I'm not really familiar with this new format yet.)
```
@type {('translucent'|'lightTranslucent'|'transparent')}
```

Reply:
I've never tried without and the examples I've seen (and used) all have the parens.

Reply:
It still works, but fixed the typo and added the parens.